packages/insomnia/src/ui/components/modals/settings-modal.tsx

```diff
@@ -19,128 +17,95 @@ import { ImportExport } from '../settings/import-export';
-              <ImportExport
-                hideSettingsModal={this.hide}
-              />
-            </TabPanel>
-            <TabPanel className="react-tabs__tab-panel pad scrollable">
-              <ThemePanel />
-            </TabPanel>
-            <TabPanel className="react-tabs__tab-panel pad scrollable">
-              <Shortcuts
-                handleUpdateKeyBindings={this._handleUpdateKeyBindings}
-              />
-            </TabPanel>
-            <TabPanel className="react-tabs__tab-panel pad scrollable">
-              <Account />
-            </TabPanel>
-            <TabPanel className="react-tabs__tab-panel pad scrollable">
-              <Plugins settings={settings} />
-            </TabPanel>
-          </Tabs>
-        </ModalBody>
-      </Modal>
-    );
-  }
-}
-
-export const showSettingsModal = () => showModal(SettingsModal);
-
-const mapStateToProps = (state: RootState) => ({
-  settings: selectSettings(state),
+          {email ? ` – ${email}` : null}
+        </span>
+      </ModalHeader>
+      <ModalBody noScroll>
+        <Tabs className="react-tabs" defaultIndex={currentTabIndex ?? undefined}>
+          <TabList>
+            <Tab tabIndex="-1">
+              <Button value="General">General</Button>
+            </Tab>
+            <Tab tabIndex="-1">
+              <Button value="Import/Export">Data</Button>
+            </Tab>
+            <Tab tabIndex="-1">
+              <Button value="Themes">Themes</Button>
+            </Tab>
+            <Tab tabIndex="-1">
+              <Button value="Shortcuts">Keyboard</Button>
+            </Tab>
+            <Tab tabIndex="-1">
+              <Button value="Account">Account</Button>
+            </Tab>
+            <Tab tabIndex="-1">
+              <Button value="Plugins">Plugins</Button>
+            </Tab>
+          </TabList>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <General />
+          </TabPanel>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <ImportExport hideSettingsModal={() => modalRef.current?.hide()}/>
+          </TabPanel>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <ThemePanel />
+          </TabPanel>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <Shortcuts
+              handleUpdateKeyBindings={handleUpdateKeyBindings}
+            />
+          </TabPanel>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <Account />
+          </TabPanel>
+          <TabPanel className="react-tabs__tab-panel pad scrollable">
+            <Plugins settings={settings} />
+          </TabPanel>
+        </Tabs>
+      </ModalBody>
+    </Modal>
+  );
 });
-
-export const SettingsModal = connect(
-  mapStateToProps,
-  null,
-  null,
-  { forwardRef: true },
-)(UnconnectedSettingsModal);
+SettingsModal.displayName = SETTINGS_MODAL_DISPLAYNAME;
```

Comment:
```suggestion
SettingsModal.displayName = SETTINGS_MODAL_DISPLAY_NAME;
```

In case you agree with the above suggestion :P